scripts/setup/restore-backup

```diff
@@ -97,7 +102,7 @@ def restore_backup(tarball_file):
                 "zulip_base",
             ]
         )
-        as_postgres = ["su", "-s", "/usr/bin/env", "-", "--", POSTGRES_USER]
+        as_postgres = ["su", "-p", "-s", "/usr/bin/env", "-", "--", POSTGRES_USER]
```

Comment:
`su -p` is dangerous; just pass these args as arguments to `usr/bin/env`.  `man env` but basically the whole point of `env` is so that you can do `env PGHOST=%s ...` to set environment variables.